snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -14,6 +14,71 @@ init_config:
 ## 
 #  ignore_nonincreasing_oid: false
 
+## @param metrics - list of elements - required
+## Specify metrics you want to monitor by using MIBS for Counter and Gauge, i.e:
```

Comment:
I'd mention somewhere that these are exactly the same parameters as the one you define in an instance, but that they apply to every defined instance and can be overridden per instance.